### ISSUE TITLE
Toolspecifics Mapet

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ebouhoubeiny/cpacs#387]
```
Xpath of the element CPACS:
"/cpacs/aircraft/..."

Type in CPACS:
"aircraftModelType"

What version of CPACS are you using?
"CPACS 1.7"

Please specifiy further information for your issue:


Please provide any additional information below:


```

Original issue reported on code.google.com by `daniel.b...@googlemail.com` on 9 Jul 2014 at 8:16

**Comment by GoogleCodeExporter** [comment]
```
    <MAPET2C>
      <tool>
        <name>MAPET2C</name>
        <version>V1.0</version>
      </tool>
      <aircraftModelUID>D150_VAMP</aircraftModelUID>
      <aircraftFlightCaseUID>D150_VAMP_FlightCaseNominal</aircraftFlightCaseUID>
      <toolParameters>
        <analysisCase>1</analysisCase>
        <aircraftConfiguration>1</aircraftConfiguration>
        <Case1>
          <doStabTrim>0</doStabTrim>
          <estimatedV2>90</estimatedV2>
          <thetaV2>18</thetaV2>
          <V_IAS_rot_init mapType="vector">50; 54; 58; 62; 66;</V_IAS_rot_init>
          <Ctrl>
            <P_elev>0.065</P_elev>
            <I_elev>0.01</I_elev>
            <D_elev>0</D_elev>
            <P_q>-1.5</P_q>
            <I_q>0</I_q>
            <D_q>0</D_q>
            <theta1>15</theta1>
            <theta2>18</theta2>
          </Ctrl>
        </Case1>
        <Case2>
          <V_IAS_engFail mapType="vector">35; 40; 45; 45</V_IAS_engFail>
          <maxLatDev>10</maxLatDev>
          <Ctrl>
            <P>0.02</P>
            <I>0.001</I>
            <D>0.0</D>
          </Ctrl>
        </Case2>
        <Case3>
          <max_TO_RWY>1000</max_TO_RWY>
          <V_IAS_engFail mapType="vector">60;65;70;75;80</V_IAS_engFail>
          <V_NG_limit>65</V_NG_limit>
          <maxBrake>1</maxBrake>
          <Ctrl>
            <P_rud>0.02</P_rud>
            <I_rud>0.001</I_rud>
            <D_rud>0.0</D_rud>
            <P_NG>0.02</P_NG>
            <I_NG>0</I_NG>
            <D_NG>0</D_NG>
          </Ctrl>
        </Case3>
        <Case4>
          <V_IAS_LD>95</V_IAS_LD>
          <RWYCond>0</RWYCond>
          <useReverser>1</useReverser>
          <useSpoiler>1</useSpoiler>
          <approachLength>1500</approachLength>
          <flareHeight>15.24</flareHeight>
          <FPA_TouchDown>-1</FPA_TouchDown>
          <FPA_StartApp>-2.5</FPA_StartApp>
          <r_CG_MLDG>3</r_CG_MLDG>
          <speedReverserOff>30</speedReverserOff>
          <maxReverser>-0.8</maxReverser>
          <maxBrake>1</maxBrake>
          <Ctrl>
            <EngRateLim>-0.1</EngRateLim>
            <P_elev>0.065</P_elev>
            <I_elev>0.01</I_elev>
            <D_elev>0.0</D_elev>
            <P_q>-1.5</P_q>
            <I_q>0</I_q>
            <D_q>0.0</D_q>
            <P_thrust>0.0</P_thrust>
            <I_thrust>0.0</I_thrust>
            <D_thrust>0.0</D_thrust>
          </Ctrl>
        </Case4>
      </toolParameters>
    </MAPET2C>

```

Original comment by `daniel.b...@googlemail.com` on 9 Jul 2014 at 8:17
- Changed title: **Toolspecifics Mapet**
- Changed state: **Accepted**
- Added labels: **milestone-CPACS2.2.1**

**Comment by GoogleCodeExporter** [comment]
```
Implemented for CPACS 2.2.1
```

Original comment by `daniel.b...@googlemail.com` on 27 Jan 2015 at 2:06
- Changed state: **Done**
